src/components/message/message.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+import messageDemo from './demo/default-message.twig';
+import messageWithCloseDemo from './demo/message-with-close.twig';
+import './message.scss';
+
+<!-- @TODO Generate an example of a message with a decorative icon. -->
```

Comment:
If this is more of a feature than a code change, it might be better as a new issue or a todo on an existing issue?

Reply:
I decided to add a new issue: 
https://github.com/cloudfour/cloudfour.com-patterns/issues/684